bin/i18n/sync-codeorg-out

```diff
@@ -3,7 +3,7 @@
 # Distribute downloaded translations from i18n/locales
 # back to blockly-core, apps, pegasus, and dashboard.
 
-require File.expand_path('../../pegasus/src/env', __FILE__)
+require File.expand_path('../../../pegasus/src/env', __FILE__)
```

Comment:
Asher recently moved my scripts into bin/i18n instead of bin/ so I'll have to update this for all of them in my future PR tonight when I do a full sync. Is there a better way I should be including this so I don't have to update each time we move them?